cmd/convert.go

```diff
@@ -101,9 +103,11 @@ func init() {
 
 	// OpenShift only
 	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", true, "Generate an OpenShift deploymentconfig object")
+	convertCmd.Flags().BoolVar(&ConvertInsecureRepo, "insecure-repository", false, "Specify to use insecure docker repository while generating Openshift image stream object")
```

Comment:
Hi @qujinping this description may be better:
`Use an insecure Docker repository for OpenShift ImageStream`